### DISCUSSION TITLE
How can I ignore or mute specific Issues or Issue types in Security Advisor such the 'Security Definer' error?

### DISCUSSION THREAD

**Discussion by RyanParkGH** [supabase/supabase#28608] [Questions]
Security Advisor is a great tool for spotting deficiencies in my Supabase configuration. However, it appears to take an absolutist stance on some issues which are not black and white.

For instance. I have a database with RLS set, yet, I need anon to have access to a variety of tables' columns via a public view. I do not want anon to have access to any of the underlying tables, just columns contained in the view.

The only way to do this is to create a view with the Security Definer property.

I create a role with minimal permissions 'anon_viewer', add the anon_viewer role to each underlying table with a read-only (SELECT) policy, and make it the owner of the public view.

This is a safe approach.

Yet, I will continue to get errors about security definer indefinitely via email and via a red dot on Security Advisor in the dashboard. This negates the effectiveness of the Security Advisor function, and it begs the question whether using security definer should actually be an error in Security Advisor, versus a warning  or info level issue.

If I could acknowledge the choice I've made to consciously use security definer, by muting or ignoring certain issues, Security Advisor would remain useful. How can I make this possible?

Thank you.

**Comment by kawaijoe** [comment]
From my understanding, this isn't possible. I'm currently facing this issue as well.

It could be better to convert this into a `feature request` instead of a `question`.

**Reply by RyanParkGH** [reply]
Thank you, I already did submit feedback in the supabase dashboard which has been acknowledged. So I will leave it at that. :)
